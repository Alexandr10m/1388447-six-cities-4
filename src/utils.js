const firstWordInUpper = (str) => {
  const isString = typeof str === `string` || !!str;

  return isString ? str[0].toUpperCase() + str.slice(1) : str;
};

export {firstWordInUpper};
