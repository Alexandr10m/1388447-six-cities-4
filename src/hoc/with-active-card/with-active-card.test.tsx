import * as React from "react";
import * as renderer from "react-test-renderer";
import withActiveCard from "./with-active-card";
import {noop} from "../../utils";


const MockComponent = () => <div/>;

const MockComponentWrapped = withActiveCard(MockComponent);

describe(`Snapshot of withActiveCards`, () => {
  it(`withActiveCards should render correctly`, () => {
    const tree = renderer
      .create(
          <MockComponentWrapped
            onActiveCard={noop}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
