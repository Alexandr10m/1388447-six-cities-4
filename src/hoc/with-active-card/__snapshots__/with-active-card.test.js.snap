// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of withActiveCards withActiveCards should render correctly 1`] = `<div />`;
